docs/MapColonies/Raster/services/catalog/profile_v1_0.md

```diff
@@ -15,37 +15,90 @@ tags:
+
+<details>
+  <summary>mc:MCRasterRecord response example</summary>
+
+  ``` xml title="mc:MCRasterRecord"
+    <mc:MCRasterRecord>
+        <mc:classification>6</mc:classification>
+        <mc:creationDateUTC>2023-11-06T23:51:11Z</mc:creationDateUTC>
+        <mc:description>test desription</mc:description>
+        <mc:footprint>{"type":"Polygon","coordinates":[[[34.388709841231474,31.435415286905005],[34.606444758516666,31.435415286905005],[34.606444758516666,31.623449225481167],[34.388709841231474,31.623449225481167],[34.388709841231474,31.435415286905005]]]}</mc:footprint>
+        <mc:id>045eaa61-8f61-48d3-a240-4b02a683eca3</mc:id>
+        <mc:ingestionDate>2024-10-05T08:49:54Z</mc:ingestionDate>
+        <mc:insertDate>2023-11-06T23:51:11Z</mc:insertDate>
+        <mc:links scheme="WMS" name="teset_north-Orthophoto" description="">https://raster-mapproxy-int-nginx-route/api/raster/v1/service?REQUEST=GetCapabilities</mc:links>
+        <mc:links scheme="WMS_BASE" name="teset_north-Orthophoto" description="">https://raster-mapproxy-int-nginx-route/api/raster/v1/wms</mc:links>
+        <mc:links scheme="WMTS" name="teset_north-Orthophoto" description="">https://raster-mapproxy-int-nginx-route/api/raster/v1/wmts/1.0.0/WMTSCapabilities.xml</mc:links>
+        <mc:links scheme="WMTS_KVP" name="teset_north-Orthophoto" description="">https://raster-mapproxy-int-nginx-route/api/raster/v1/service?REQUEST=GetCapabilities&amp;SERVICE=WMTS</mc:links>
+        <mc:links scheme="WMTS_BASE" name="teset_north-Orthophoto" description="">https://raster-mapproxy-int-nginx-route/api/raster/v1/wmts</mc:links>
+        <mc:maxResolutionDeg>0.072</mc:maxResolutionDeg>
+        <mc:maxResolutionMeter>0.15</mc:maxResolutionMeter>
+        <mc:minHorizontalAccuracyCE90>50.0</mc:minHorizontalAccuracyCE90>
+        <mc:producerName>IDFMU</mc:producerName>
+        <mc:productBBox>34.388709841231474,31.435415286905005,34.606444758516666,31.623449225481167</mc:productBBox>
+        <mc:productId>teset_north</mc:productId>
+        <mc:productName>test noth</mc:productName>
+        <mc:productType>Orthophoto</mc:productType>
+        <mc:productVersion>2.0</mc:productVersion>
+        <mc:region>test</mc:region>
+        <mc:sensors>UNIDENTIFIED,FEATURE</mc:sensors>
+        <mc:imagingTimeEndUTC>2023-11-05T09:31:00Z</mc:imagingTimeEndUTC>
+        <mc:imagingTimeBeginUTC>2023-11-01T09:31:00Z</mc:imagingTimeBeginUTC>
+        <mc:SRS>4326</mc:SRS>
+        <mc:SRSName>WGS84GEO</mc:SRSName>
+        <mc:transparency>OPAQUE</mc:transparency>
+        <mc:type>RECORD_RASTER</mc:type>
+        <mc:updateDateUTC>2024-04-10T08:02:01Z</mc:updateDateUTC>
+        <ows:BoundingBox crs="urn:x-ogc:def:crs:EPSG:6.11:4326" dimensions="2">
+            <ows:LowerCorner>31.435415286905005 34.388709841231474</ows:LowerCorner>
+            <ows:UpperCorner>31.623449225481167 34.606444758516666</ows:UpperCorner>
+        </ows:BoundingBox>
+    </mc:MCRasterRecord>
+  ```
+</details>
```

Comment:
I don't think we want this here... There are examples in the `Getting Started` guide. This should only have details about the `profile`.

Reply:
you want me to delete it?